src/hooks-log.coffee

```diff
@@ -0,0 +1,23 @@
+util = require 'util'
+
+hooksLog = (logs = [], logger, logVariant, content) ->
+  if arguments.length is 4 and logVariant in ['info', 'debug', 'warn', 'verbose', 'error', 'log']
```

Comment:
It allows you to have logs more verbose, if you want to, according to the log-level you are using.
Because it uses the same `logger` as the rest of Dredd, it actually __is usable__ when used as
`hooks.log('verbose', 'Just starting validations')`